flocker/restapi/docs/publicapi.py

```diff
@@ -296,6 +296,7 @@ def _formatRouteBody(data, schema_store):
     """
     baseSubstitutions = {
         u"DOMAIN": u"example.com",
+        u"NODE_0": u"node000.example.com",
```

Comment:
:exclamation: This should be an IP address, otherwise, the examples should fail validation.